rs/sns/governance/src/governance.rs

```diff
@@ -3939,6 +3949,48 @@ impl Governance {
         remove_neuron_from_follower_index(&mut self.topic_follower_index, neuron);
         add_neuron_to_follower_index(&mut self.topic_follower_index, neuron);
 
+        // Fourth, remove any legacy following (based on individual proposal types under the topics
+        // that were modified by this command).
+        for topic in &mentioned_topics {
+            for function in GovernanceProto::get_custom_functions_for_topic(
+                &self.proto.id_to_nervous_system_functions,
+                *topic,
+            ) {
+                legacy::remove_neuron_from_function_followee_index_for_function(
```

Comment:
Does this also remove the configuration from the neuron itself?

Reply:
Good catch, fixed it.